docs/DeveloperGuide.md

```diff
@@ -4,30 +4,274 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
```

Comment:
Perhaps can consider removing the acknowledgements section if you don't have sources to acknowledge

Reply:
Noted, thanks for pointing this out!